invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides

```diff
@@ -109,6 +110,13 @@
     }
 
     .content {
+      .description.truncate-lines-2 {
```

Comment:
Looks like [not very well supported](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp) by browsers (yet)? Did you manage to test it with the mainstream browsers?

Reply:
I checked https://caniuse.com/css-line-clamp it does not seem to be aligned with mozilla docs

Reply:
![Screenshot 2024-10-10 at 13 54 29](https://github.com/user-attachments/assets/9105f909-ed8a-4fac-a365-f273410c9905)

Reply:
checked on safari, chrome, firefox, opera: works on all of them
mobile: chrome, firefox, duduckgo, safari: works as well. Looks like the docs are not up to date